lib/src/game/game_widget.dart

```diff
@@ -423,65 +425,72 @@ Widget _applyBasicGesturesDetectors(Game game, Widget child) {
 
 Widget _applyAdvancedGesturesDetectors(Game game, Widget child) {
   final Map<Type, GestureRecognizerFactory> gestures = {};
```

Comment:
I dont know why I decided to refactor this, I was not fully understanding it and I really disliked the fact that I had to declare yet another "fake interface" with the whole set of onX methods. In the end I think this is not particularly better than the version before but I don't think it's worse either so I left. I like that this version makes it more clear that there can only really be one handler. the list might trick you into thinking there might be two.